tests/_files/source_with_multiline_constant_return.php

```diff
@@ -485,10 +485,30 @@ public function unaryMinusNowdoc(): bool
     {
         return
             -
+            (int)
```

Comment:
no cast needed here too, and the type can be changed to `int`

Reply:
`Uncaught TypeError: Unsupported operand types: string * int`

Reply:
Without the typecast, it doesn't compile